pkg/expapi/types.go

```diff
@@ -361,3 +361,34 @@ type ThirdPartyResourceDataList struct {
 	// Items is a list of third party objects
 	Items []ThirdPartyResourceData `json:"items"`
 }
+
+// LimitCapability defines the privilege and capability limits of PodSpecs in a namespace
```

Comment:
@liggitt the I am not sure part was related to not being sure if we were
going to move build pods into a separate namespace in future.

On Friday, September 4, 2015, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/expapi/types.go
> <https://github.com/kubernetes/kubernetes/pull/13582#discussion_r38776056>
> :
>
> > @@ -361,3 +361,34 @@ type ThirdPartyResourceDataList struct {
> >  	// Items is a list of third party objects
> >  	Items []ThirdPartyResourceData `json:"items"`
> >  }
> > +
> > +// LimitCapability defines the privilege and capability limits of PodSpecs in a namespace
>
> we already have the concept of users in the authenticator/authorizer
> interfaces... see main thread for discussion about namespace+admission vs
> user+policy application of this, and the intersections of those cases
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13582/files#r38776056>.
>